ecs-init/docker/dependencies.go

```diff
@@ -22,8 +22,8 @@ import (
 	"net/url"
 	"time"
 
-	"github.com/aws/amazon-ecs-init/ecs-init/backoff"
-	"github.com/aws/amazon-ecs-init/ecs-init/config"
+	"github.com/aws/amazon-ecs-agent/ecs-init/backoff"
```

Comment:
I think this is now a duplicated package since agent also has a backoff package. Should we use that instead?

Reply:
Good point! In these merged branches there is an opportunity to pull out shared functionality. In this case now though I don't think it makes sense, primarily since these are the legacy branches so I'm trying to avoid changing the code itself. Also, we're currently building init and agent separately and trying to get that to work, would need to do some more considerable refactoring I think to build shared code (though haven't looked into this too thoroughly yet)